### DISCUSSION TITLE
dependabot wants to "upgrade" beta.406 to beta.66 (stylecop analyzers package)

### DISCUSSION THREAD

**Discussion by fretje** [community/community#12698] [Code Security]
Not sure if this is an issue with dependabot or with the package in question (stylecop analyzers)... 

But see this PR created by dependabot:

https://github.com/fullstackhero/blazor-wasm-boilerplate/pull/158

it wants to "upgrade" 1.2.0-beta.406 to 1.2.0-beta.66 

seems more like a "downgrade" to me?

**Comment by jakecoffman** [comment]
This was a bug in dependabot-core which we just fixed. More info here: https://github.com/dependabot/dependabot-core/issues/1972#issuecomment-1067165556
